qhub/template/{{ cookiecutter.repo_directory }}/image/jupyterhub/environment.yaml

```diff
@@ -0,0 +1,12 @@
+name: base
+channels:
+  - conda-forge
+dependencies:
+ - pip
+ - jupyterhub >= 1.3.0
+ - bcrypt
+ - oauthenticator
+ - escapism
```

Comment:
What is this required for?

Reply:
https://github.com/Quansight/qhub-cloud/blob/59680e03f5c8132c40926b9d13fa7b0590796e09/qhub/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/infrastructure/jupyterhub.yaml#L113

Reply:
I think it is included with jupyterhub but wanted to explicitly mention the dependency